modules/nw-udn-benefits.adoc

```diff
@@ -29,4 +29,9 @@ User-defined networks provide the following benefits:
 +
 * **Network parity**: With user-defined networking, the migration of applications from OpenStack to {product-title} is simplified by providing similar network isolation and configuration options.
 
-Developers and administrators can create a user-defined network that is namespace scoped using the custom resource. An overview of the process is: create a namespace, create and configure the custom resource, create pods in the namespace.
\ No newline at end of file
+Developers and administrators can create a user-defined network that is namespace scoped using the custom resource. An overview of the process is as follows:
++
+. An administrator creates a namespace for a user-defined network with the `k8s.ovn.org/primary-user-defined-network` label. 
+. Optional: An administrator grants a non-administrator user `admin` privileges on the user-defined network namespace.
```

Comment:
I think we can ommit this, we have added ClusterRoles aggregated to the OCP view and edit ClusterRole so that non-amin users could read & modify UDN CRs with no additional configurations.
Please see https://github.com/openshift/cluster-network-operator/pull/2627

Reply:
Removed ". Optional: An administrator grants a non-administrator user `admin` privileges on the user-defined network namespace."

Reply:
Removed.